pkgs/tools/networking/ppp/default.nix

```diff
@@ -34,7 +34,9 @@ stdenv.mkDerivation rec {
   '';
 
   postFixup = ''
-    substituteInPlace $out/bin/{pon,poff,plog} --replace "/usr/sbin" "$out/bin"
+    for tgt in pon poff plog; do
+      substituteInPlace "$out/bin/$tgt" --replace "/usr/sbin" "$out/bin"
```

Comment:
AFAIK, `substituteInPlace` has never accepted multiple targets. At least, that is not the documented behavior. I don't know how this ever worked!